### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'accessibilityData')

### ISSUE THREAD

**Issue by MuhammadRestu999** [BochilTeam/scraper#38]
Beberapa query bikin error gini

```
TypeError: Cannot read properties of undefined (reading 'accessibilityData')
    at file:///home/MuhammadRestu/Rain-BotV2/node_modules/@bochilteam/scraper/lib/esm/social-media/youtube-search.js:71:122
    at Array.forEach (<anonymous>)
    at youtubeSearch (file:///home/MuhammadRestu/Rain-BotV2/node_modules/@bochilteam/scraper/lib/esm/social-media/youtube-search.js:24:71)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.handler (file:///home/MuhammadRestu/Rain-BotV2/plugins/youtube-play.js:10:14)
    at async Object.handler (file:///home/MuhammadRestu/Rain-BotV2/handler.js?update=1670815978468:716:21)
```

**Comment by MuhammadRestu999** [comment]
workas pas add `_2?.accessibility?.accessibilityData?.label` di line 71:122 lib/esm/social-media/youtube-search.js dan `_8.runs?.[0]` di line 85:117 lib/esm/social-media/youtube-search.js
